etcd/cmd/microshift-etcd/run.go

```diff
@@ -1,22 +1,38 @@
 package main
 
 import (
+	"context"
 	"fmt"
+	"math"
 	"net"
 	"net/url"
 	"os"
 	"os/signal"
 	"path/filepath"
 	"syscall"
+	"time"
 
 	"github.com/openshift/microshift/pkg/config"
 	"github.com/openshift/microshift/pkg/util/cryptomaterial"
 
 	"github.com/spf13/cobra"
 	etcd "go.etcd.io/etcd/server/v3/embed"
+	"go.etcd.io/etcd/server/v3/mvcc/backend"
 	"k8s.io/klog/v2"
 )
 
+// TODO these need to be configurable (with an option to completely turn off)
```

Comment:
I think these should be user configurable so if they don't want/need defrag, they can control how and when it runs.

Reply:
I added the configuration to the Microshift config so the microshift-etcd process itself consumes the config.